frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1558,6 +1625,13 @@ impl<T: Config> Pallet<T> {
 				if Self::round() != 1 {
 					log!(info, "Finalized election round with compute {:?}.", compute);
 				}
+				match Self::try_update_minimum_untrusted_score(score) {
+					Ok(_) => (),
+					Err(e) => {
+						defensive!("minimum score update: {:?}", e);
```

Comment:
```suggestion
						defensive!("could not update minimum trusted score: {:?}", e);
```      
     
         
The match statement could also be potentially rewritten as following?
`Self::try_update_minimum_untrusted_score(score).defensive()`